docs/reference/release-notes/860.md

```diff
@@ -11,19 +11,136 @@ keywords:
+
+### Data viewer modeling interface<span class="badge badge--long" title="This feature affects Modeler">Modeler</span>
+
+<!-- https://github.com/camunda/product-hub/issues/2011 -->
+
+You can use the new **variables view** to track variables defined in a process diagram.
+
+- This view shows you where a variable was created, which elements write to the variable, and the variable’s scope.
+- You can use this view to see which variables are available for use in an element, and to troubleshoot problems with variables.
+
+### Get started with microservice orchestration <span class="badge badge--long" title="This feature affects Help Center">Help Center</span>
+
+<!-- https://github.com/camunda/product-hub/issues/2463 -->
+
+The [Help Center](/docs/reference/help-center.md) now features a step-by-step guide to complete our microservice orchestration tutorial.
+
+### Get started with human task orchestration <span class="badge badge--long" title="This feature affects Help Center">Help Center</span> <span class="badge badge--long" title="This feature affects Tasklist">Tasklist</span>
+
+<!-- https://github.com/camunda/product-hub/issues/2395 -->
+
+The [Help Center](/docs/reference/help-center.md) now features a step-by-step guide to complete our human task orchestration (HTO) tutorial, explaining the critical features that every developer interested in human task orchestration needs to know.
+
+- Users get a glimpse into the user experience for at-scale adoption.
+- Enterprise architects get a picture of how Camunda's out-of-the-box Tasklist fits into their architecture.
+
+### Internationalization (I18n) for Tasklist <span class="badge badge--long" title="This feature affects Tasklist">Tasklist</span>
+
+<!-- https://github.com/camunda/product-hub/issues/2378 -->
+
+When creating a process for a local (non-English) region, you can design forms in the local language and have the Tasklist interface in the same language.
+
+- By default, Tasklist supports four languages: English, French, German, and Spanish.
+- The interface language is automatically determined by the user device's language settings.
+- The interface defaults to English if the device's language is not supported.
+
+### Message correlation with result <span class="badge badge--long" title="This feature affects Zeebe">Zeebe</span>
+
+<!-- https://github.com/camunda/product-hub/issues/1818 -->
+
+Real-time feedback is added for message correlation for messages with `ttl=o`, enabling external systems to immediately determine the success or failure of message correlation. This enhancement allows external systems to take prompt and appropriate actions based on the correlation result, improving overall efficiency and reducing response times.
+
+### Public Marketplace Blueprint support for HTO & DMN <span class="badge badge--long" title="This feature affects Web Modeler">Web Modeler</span> <span class="badge badge--long" title="This feature affects Marketplace">Marketplace</span>
```

Comment:
@YanaSegal have we decided to capitalize Blueprints for a particular reason here or is there a related thread? IMO this should be lowercase, but open to opinions. CC @camunda/tech-writers

Reply:
@christinaausley 
I think there is no reason for capitalizing blueprint - I can't recall any decision to capitalize it.